bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/MessageEvent.kt

```diff
@@ -0,0 +1,6 @@
+package com.bugsnag.android
+
+class MessageEvent(
```

Comment:
Some javadoc here would be good to give context.

Reply:
Added JavaDoc for this interface